src/data/finset/basic.lean

```diff
@@ -1301,33 +1201,33 @@ by simp [finset.nonempty]
 by simpa [eq_empty_iff_forall_not_mem]
 
 /-! ### piecewise -/
+
 section piecewise
 
 /-- `s.piecewise f g` is the function equal to `f` on the finset `s`, and to `g` on its
 complement. -/
-def piecewise {α : Type*} {δ : α → Sort*} (s : finset α) (f g : Πi, δ i) [∀j, decidable (j ∈ s)] :
-  Πi, δ i :=
+def piecewise {α : Type*} {δ : α → Sort*} (s : finset α) (f g : Π i, δ i) [∀ j, decidable (j ∈ s)] :
```

Comment:
If we're nit-picking spacing, we may as well also nitpick `Π` vs `∀` in the same diff: decidable is data so should be the former
```suggestion
def piecewise {α : Type*} {δ : α → Sort*} (s : finset α) (f g : Π i, δ i) [Π j, decidable (j ∈ s)] :
```